provider/cluster/kube/debug/ns-netpol.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: v1
```

Comment:
accidental commit?  put in `_docs` somewhere if not, with a comment at the top explaining what it is.